ocp_cleanup.sh

```diff
@@ -8,6 +8,8 @@ source ocp_install_env.sh
 sudo systemctl stop fix_certs.timer
 systemctl is-failed fix_certs.service >/dev/null && sudo systemctl reset-failed fix_certs.service
 
+sudo podman stop machine-api-operator || echo "No local machine-api-operator running"
```

Comment:
Should it just be stopped, or rm'd as well?

Reply:
It's created with --rm so IIUC stopping it will cause it to delete itself.

Reply:
It does not if there was a problem starting the container, we've run into issues with not explicitly removing containers during clean.